packages/esm-patient-immunizations-app/src/immunizations/immunizations.resource.tsx

```diff
@@ -43,13 +43,13 @@ export async function getImmunizationsConceptSet(
   return result;
 }
 
-export function useImmunizationsConceptSet() {
+export function useImmunizationsConceptSet(customRepresentation = 'full') {
```

Comment:
Maybe instead of taking this as an argument, you can just hard-code the custom representation here? After all, this hook is only used in the code you're using (nice thing about MFs is that they are small, so you can pretty much modify the parts without worrying that someone else is using it).

Reply:
I agree with @ibacher, we don't need to pass the custom string as an argument, until and unless different payload is required at 2 or more places.,